### ISSUE TITLE
QA: [mobile] offline mode with expired OIDC access token

### ISSUE THREAD

**Issue by chibenwa** [linagora/tmail-flutter#2966]
### Description

Expectation: 

**GIVEN** network failure (connect timeout, etc...)
**AND** expired OIDC access token
**WHEN** user performs action (read cached / not cached mails, send mails in sending queue)
**THEN** user is not logged out and actions are  done as if tokens were not expired.

Notes: beware of regression!
 - Still refresh access token automatically
 -  **401** should bring to login page 
 - etc...

**Comment by hoangdat** [comment]
- [x] For mobile: no problem with OIDC in offline mode

**Comment by chibenwa** [comment]
After expirations of the access token?

**Comment by hoangdat** [comment]
yes, no problem of token expiration in offline mode (after offline mode too)
